Sources/Fullscreen/EmptyView/EmptyView.swift

```diff
@@ -163,6 +169,11 @@ public class EmptyView: UIView {
     private func setup() {
         backgroundColor = .milk
 
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(userHasTapped))
+        addGestureRecognizer(tapGesture)
+
+        becomeFirstResponder()
```

Comment:
Just a question, why are you calling `becomeFirstResonder()` here?

Reply:
Ah, ok. Just so you know, it does say in Apples documentation that you should 
[_never call this method on a view that is not part of an active view hierarchy_](https://developer.apple.com/documentation/uikit/uiresponder/1621113-becomefirstresponder).
So maybe you should move this call to after you've added the view as a subview? I'm sure everything works as it should now though, but it might cause unexpected behaviour.

Reply:
I just read that myself. 😅Not entirely sure what it means, but I could move it to after the adding of subviews yes. Also, I can guard the `window` property in the `canBecomeFirstResponder` override so that it returns `false` if `window == nil`.

Reply:
Calling `becomefirstresponder` on the view after adding its subview wouldn't make any difference. You'd have to wait for calling it after the view itself has been added as a subview to a view thats in an active view hierarchy.